Dockerfile

```diff
@@ -4,3 +4,13 @@ MAINTAINER support@68publishers.io
 
 ########################################################################################################################
 FROM postgres:13.6-alpine AS db
+
+########################################################################################################################
+FROM 68publishers/docker-images:php-nginx-unit-7.4 AS worker
+
+RUN apk add --update --no-cache --allow-untrusted supervisor
+RUN mkdir -p "/etc/supervisor/logs"
+
+COPY docker/supervisor/supervisord.conf /etc/supervisor/supervisord.conf
+
+CMD ["/usr/bin/supervisord", "-n", "-c",  "/etc/supervisor/supervisord.conf"]
```

Comment:
There is some solution thats wrapped the supervisor into the bash script. Should we use it that way? https://debricked.com/blog/scaling-symfony-consumers-using-kubernetes/